starwars/src/App.js

```diff
@@ -25,11 +25,23 @@ class App extends Component {
   }
   render() {
     return (
+      {this.state.starwarsChars.map( (element, index) =>
```

Comment:
Here you're only rendering each character's height attribute. Try rendering all of their attributes.